src/Control/Distributed/Process/Node.hs

```diff
@@ -661,6 +661,30 @@ instance Show ProcessKillException where
   show (ProcessKillException pid reason) =
     "killed-by=" ++ show pid ++ ",reason=" ++ reason
 
+ncSendToProcess :: ProcessId -> Message -> NC ()
+ncSendToProcess pid msg = do
+    node <- ask
+    if processNodeId pid == localNodeId node
+      then ncEffectLocalSend node pid msg
```

Comment:
hm, this line https://github.com/haskell-distributed/distributed-process/pull/264/files#diff-8684dec0b32dae52a61ac233d229266cL521
should be move here.